l10n/en/firefox/browsers/compare/safari.ftl

```diff
@@ -68,7 +72,7 @@ compare-safari-both-browsers-have-a-lot = Both browsers have a lot of crossover
 
 # Variables:
 #   $attrs (string) - link to /firefox/accounts/ with additional attributes for analytics
-compare-safari-firefox-also-offers-a-similar = { -brand-name-firefox } also offers a similar sync feature when you sign up for a free <a { $attrs }>{ -brand-name-firefox-account }</a> that enables users to easily synchronize their bookmarks, browsing history, preferences, passwords, filled forms, add-ons, and the last 25 opened tabs across multiple computers. What sets { -brand-name-firefox } apart from { -brand-name-safari } is that it is available on any desktop or mobile platform, { -brand-name-ios }, { -brand-name-android }, { -brand-name-windows } or { -brand-name-macos }, boosting its portability across any device you may own.
+compare-safari-firefox-also-offers-a-similar = { -brand-name-firefox } also offers a similar sync feature when you sign up for a free <a { $attrs }>{ -brand-name-firefox-account }</a> that enables users to easily synchronize their bookmarks, browsing history, preferences, passwords, filled forms, add-ons, and the last 25 opened tabs across multiple computers. What sets { -brand-name-firefox } apart from { -brand-name-safari } is that it is available on any desktop or mobile platform, { -brand-name-ios }, { -brand-name-android }, { -brand-name-windows } or { -brand-name-mac }, boosting its portability across any device you may own.
```

Comment:
This is a bug fix I never noticed until making this update, changing `-brand-name-macos` (which doesn't exist in brands.ftl) to `-brand-name-mac` (which does). I don't think fixing that reference actually makes this a new string... but if it does I can turn this into a new string and mark the old one as obsolete.